optuna/integration/wandb.py

```diff
@@ -95,10 +104,22 @@ def __init__(
                 )
             )
 
-        self._metric_name = metric_name
         self._wandb_kwargs = wandb_kwargs or {}
+        tags = self._wandb_kwargs.get("tags")
+
+        if as_multirun and not (
+            tags and isinstance(tags, Sequence) and all(map(lambda x: isinstance(x, str), tags))
+        ):
+            raise RuntimeError(
+                "atleast one `tag` in `wandb_kwargs` is necessary "
+                "when `as_multirun` is set to `True`."
+            )
```

Comment:
```suggestion
```

Reply:
Mmore context: We can remove this when we store the tag as in https://github.com/optuna/optuna/pull/3403#discussion_r840878666.